### PR TITLE
Feature - Wrapper API Updates: New Endpoints and Method Modifications

### PR DESCRIPTION
**Pull request by vincethecoder**
Issue List Complete as identified by @mferraco.
https://github.com/nessieisreal/nessie-javascript-wrapper/issues/13

### CODE REVIEW

File: lib/account.js [hunk 1]
```diff
@@ -39,10 +54,17 @@ define('account', function (require) {
 			});
 			return accounts;
 		},
-		getOne: function(id) {
+		/**
+		  # @Method: getAccountById
+		  # @Brief: Returns the account specified by its account ID.
+		  # @Parameters: AccountId
+		  # @Note: Accepts a string of the account ID. 
+		  # @Returns a hash with the account info.
```
Comment (mferraco) [inline lib/account.js]: these don't return a hash right?  They return a JSON object.

File: lib/bills.js [hunk 2]
```diff
@@ -9,15 +9,27 @@ define('bills', function (require) {
...
 		},
 		apiKey: function() {
 			return Config.apiKey;
 		},
-		getAllBillsByAccountId: function(accID) {
+		/**
+		  # @Method: getAllByAccountId
+		  # @Brief: Get all bills for a specific account
+		  # @Parameters: accountId
+		  # @Returns an array of hashes containing the bills.
```
Comment (mferraco) [inline lib/bills.js]: same here.  Anywhere there are hashes referenced we should remove them and update with the appropriate response value.

**Comment by mferraco** [comment]
This looks good.  Just made a few comments regarding referencing "hashes" as a return value.  Also, we should make sure we start writing some tests for this SDK, so that we know when we need to make updates (we can run the test suite and see what breaks)

**Comment by vincethecoder** [comment]
@mferraco I have updated all the endpoints to match the new SDK endpoints and the comments within the code. As far as a test suite, I have created an example file to verify that the APIs return a successful response code and expected JSON Objects. My time will not allow for a test suite for this PR. The initial thought behind this PR was to address this GIT Issue: https://github.com/nessieisreal/nessie-javascript-wrapper/issues/13. 

So a test suite will need to be created on a different PR as this one is already starting to look very bloated. In the meantime the validity of these changes can be verified by running example.html on this current branch.

**Comment by mferraco** [comment]
Sounds good to me.  Looks good man!  Thanks for putting all this work in this is awesome!  @vincethecoder  :+1: :+1: :dolphin:  <- there was no shark emoji...

**Comment by vincethecoder** [comment]
Right on!! Thanks @mferraco

**Pull request merged**